app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
Đưa toàn bộ phần này vào before_action cho anh nhé. Vs bắn thêm cái message thông báo nếu ko tìm thấy user